### DISCUSSION TITLE
How to manually change translation?

### DISCUSSION THREAD

**Discussion by felixmaker** [slint-ui/slint#3713] [Q&A]
For now, I only know how to init translations:

```
fn main() {
    slint::init_translations!(concat!(env!("CARGO_MANIFEST_DIR"), "/lang/"));
}
```

How to manually change translation? If the user wants other translation.

**Comment by ogoffart** [comment]
We haven't implemented that yet, but we have plans to do it.
I've just filled an issue to keep track of it:  https://github.com/slint-ui/slint/issues/3715
